### PR TITLE
fix(docs): correct snowflake receiver docs

### PR DESCRIPTION
**Pull request by shrutimantri**
These are some minor doc corrections.

### CODE REVIEW

File: receiver/snowflakereceiver/README.md [hunk 1]
```diff
@@ -38,11 +38,11 @@ receivers:
     password: securepassword
     account: bigbusinessaccount
     warehouse: metricWarehouse
-    collection_interval: 18m
+    collection_interval: 5m
     metrics:
       snowflake.database.bytes_scanned.avg:
         enabled: true
-      snowflake.database.bytes_deketed.avg:
+      snowflake.query.bytes_deleted.avg:
```
Comment (shrutimantri) [inline receiver/snowflakereceiver/README.md:45]: There is no metric called: `snowflake.database.bytes_deketed.avg`, and there was a spelling mistake in the word `deleted`.

**Review by songy23 — COMMENTED** [review]
> Generated code is out of date, please run "make generate" and commit the changes in this PR.

**Review by shalper2 — APPROVED** [review]

**Comment by shalper2** [comment]
lgtm thanks for catching these!

**Comment by shrutimantri** [comment]
@songy23 I have addressed the comment. Can you please review?

**Review by evan-bradley — APPROVED** [review]

**Pull request merged**